README.md

```diff
@@ -6,147 +6,86 @@
 
 **Credentials** that are created and shared by Modo. These will be different for each environment (`int`, `prod`, `local` etc...).
 
-- `APIIdentifier` - API key from Modo
-- `APISecret` - API secret from Modo
+- `api_identifier` - API key from Modo
+- `api_secret` - API secret from Modo
 
 These values will be used when intantiating the library.
 
 # Install
+Presently, the github repo is private, so you'll need to tell `go` that:
+```bash
+# add env var via `~/.zshrc` or `~/.bashrc` or the like
+export GOPRIVATE="github.com/modopayments-ux,github.com/modopayments"
+```
+Restart your terminal
+
+
 ```bash
 # in the root of the project
 go get github.com/modopayments-ux/modo2auth-go
 ```
 
-Note: This package assumes your project to be a [go module](https://blog.golang.org/using-go-modules). If it is not, you may get an error looking for a `main()` function definition.
-
 # Example Usage
 
 Here's an example using `...` to make requests. You can use your preferred method or library.
 
-## `GET` Example
-
-```go
-package main
-
-import (
-	"fmt" // for example
-	"io/ioutil" // for example
-	"net/http"
-
-	// 1. IMPORT
-	"github.com/modopayments-ux/modo2auth-go"
-)
-
-func main() {
-	// 2. INSTANTIATE - get these from MODO
-	modo := modo2auth.Config{
-		APIIdentifier: "...",
-		APISecret:     "...",
-	}
-  
-	// 3. SIGN & SEND REQUEST
-	apiHOST := "http://localhost:82"
-	apiURI := "/v2/vault/public_key"  
-	req, _ := http.NewRequest("GET", apiHOST+apiURI, nil)
-	signedReq, _ := modo.Sign(req)
-	resp, _ := http.DefaultClient.Do(signedReq)
-	readBody, _ := ioutil.ReadAll(resp.Body)
-
-	fmt.Println("Response body is", string(readBody))
-}
+## Example
 
 ```
-
-## `POST` Example
-
-```go
 package main
 
 import (
-	"fmt" // for example
-	"io/ioutil" // for example
 	"net/http"
 
-	// 1. IMPORT
-	"github.com/modopayments-ux/modo2auth-go"
+	"github.com/modopayments-ux/modo2auth_go"
 )
 
+const endpoint = "http://modopayments.com"
```

Comment:
Let's use [URL_HERE] for this const so that nobody gets the wrong idea that they'll be hitting modopayments.com for anything.

Reply:
or alternatively, use localhost:82 and add a comment about using the correct URL

Reply:
update